### ISSUE TITLE
Customize the footer to include the GitHub and Coding link of the source file

### ISSUE THREAD

**Issue by CXwudi** [CXwudi/personal-blog-source#10]
Since the repo is already open-sourced due to #7, why not include the link to this repo

Customization can be done by just copying the current HTML and add our own code. Then put the HTML codes into `[params.footer] custom` field

**Comment by CXwudi** [comment]
we can't find coding.net specific icon, let's just use a default link icon

**Comment by CXwudi** [comment]
Since 2022 May, China requires identification to open source a repo. To (possibly) avoid problems related to the content of my blog, we decide to abandon coding.net

Also, it seems like GitHub is unblocked in China. So we can close this issue for now, but if one day GitHub or China blocks access from Chinese people, we will re-open this issue or create a new issue
